Readability.js

```diff
@@ -329,10 +329,25 @@ Readability.prototype = {
   },
 
   _setNodeTag: function (node, tag) {
-    // FIXME this doesn't work on anything but JSDOMParser (ie the node's tag
-    // won't actually be set).
-    node.localName = tag.toLowerCase();
-    node.tagName = tag.toUpperCase();
+    this.log("_setNodeTag", node, tag);
+    if (node.__JSDOMParser__) {
```

Comment:
Trying to fix that by increasing the iterations massively increases the number of ops/s, which makes me think we're hitting some kind of JIT optimization, which isn't really realistic (we won't normally readerize the same document a bunch of times in a row).